mgmt/rest/v2/config.go

```diff
@@ -105,7 +139,7 @@ func (s *apiV2) deletePluginConfigItem(w http.ResponseWriter, r *http.Request, p
 		res = s.configManager.DeletePluginConfigDataNodeField(typ, name, iver, src...)
 	}
 
-	item := &PluginConfigItem{ConfigDataNode: res}
+	item := &PluginConfigItem{res}
```

Comment:
@candysmurf Will this (and all other responses) be changed to `PluginConfigResponse` in the future PR?